### PR TITLE
Update Node CHANGELOG.md

### PR DESCRIPTION
**Pull request by tdcosta100**

### CODE REVIEW

**Comment by ntadej** [comment]
Not sure this is really a feature. It's more of a bugfix actually. We were otherwise C++17 for a while now (maybe that should actually be the feature).

**Comment by tdcosta100** [comment]
Ah yes, it's not a feature, I'm just following the former notes, since a PR was approved and I forgot to mention it in the CHANGELOG.md.

**Comment by ntadej** [comment]
OK, maybe something for @louwers and @ovivoda to try to improve in the future.

**Comment by louwers** [comment]
Our main focus is Core, Android and iOS and in principle we don't spend too much time on other platforms. We also don't make releases for them.

**Comment by ntadej** [comment]
What I meant is we may want to define general guidelines on changelogs and release notes.

**Comment by wipfli** [comment]
Thanks for clarifying this @louwers. People are welcome to contribute to all platforms and it is great if they handle releases. The paid maintainers focus is on iOS and Android.

**Comment by ntadej** [comment]
Maybe I should have tagged you Oliver then 😊

**Comment by acalcutt** [comment]
Usually I do the node releases, so a appreciate the changelog being uodated sinc eit helps me make the release

I agree though that this should maybe be under bug fixes. maybe something like "fixed -std=c++11 when building node release", or something like that.

Could you add a heading 
### 🐞 Bug fixes
and add this under it, like https://github.com/maplibre/maplibre-gl-native/edit/main/CHANGELOG.md

**Comment by tdcosta100** [comment]
Done!

File: platform/node/CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,8 @@
 
 ### ✨ New features
 - *...Add new stuff here...*
+* Compile Node targets without -std=c++11 option by @tdcosta100 in https://github.com/maplibre/maplibre-gl-native/pull/926
```
Comment (acalcutt) [inline platform/node/CHANGELOG.md:6]: Could you put this one under a new heading,
```
### 🐞 Bug fixes
*...Add new stuff here...*
```

Comment (tdcosta100) [reply platform/node/CHANGELOG.md:6]: Yes, of course! Nice to have this section here.

**Review by acalcutt — APPROVED** [review]

**Pull request merged**